test/forms/AddressFormProviderSpec.scala

```diff
@@ -199,4 +206,39 @@ class AddressFormProviderSpec extends FieldBehaviours with UserAddressFixtures {
     }
   }
 
+  "for the ConsignorAddress Page" - {
+
+    lazy val gbForm = new AddressFormProvider()(ConsignorAddressPage)(dataRequest(FakeRequest(), emptyUserAnswers, ern = testGreatBritainErn))
+    lazy val xiForm = new AddressFormProvider()(ConsignorAddressPage)(dataRequest(FakeRequest(), emptyUserAnswers, ern = testErn))
```

Comment:
Could we use `testNorthernIrelandErn` instead of `testErn`

Reply:
Done in 8023662f1d217e05159c332893ce24d0532c7bb1